### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by camargo**
* **Tickets addressed:** None
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

* Add Aerie logo to top
* Add more clear list of Aerie features
* Add 'Getting Started' section
* Add 'Directory Structure' section
* Add 'Want to Help?' section
* Add 'License' section
* Remove badges (LGTM being shut down)
    * @ewferg We will revisit adding more badges in the near future 

## Verification

N/A

## Documentation

This is a doc update.

## Future work

N/A

### CODE REVIEW

**Comment by ewferg** [comment]
I like badges! Why are they going away? Is this the case for all GitHub repos?

**Comment by camargo** [comment]
@ewferg Yeah I plan on having us revisit them. The LGTM service is shutting down so that's why we are removing that one. I don't think the publish badge is super useful but I'd like to have us add our testing badges back later in some form.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
-# Aerie
-
-A suite of software tools and services created to support activity planning and sequencing needs of missions with modeling, simulation, scheduling, and validation capabilities.
+<br>
+<div align="center">
+  <img src="docs/sphinx_scylladb_theme/static/img/logos/aerie-wordmark-light.svg" height="50">
+</div>
+<br>
+
+Aerie is a software framework for modeling spacecraft. It's main features include:
```
Comment (pcrosemurgy) [inline README.md]: Small typo: "It's" should be "Its" :)

Comment (camargo) [reply README.md]: fixed

**Review by pcrosemurgy — APPROVED** [review]

**Pull request merged**